packages/core/src/Stores/ui-store.js

```diff
@@ -114,6 +114,9 @@ export default class UIStore extends BaseStore {
     @observable is_mt5_page = false;
     @observable is_nativepicker_visible = false;
 
+    // dbot
+    @observable is_bot_running = false;
```

Comment:
Please add the actions for this here. But to reappraise, can we instead provide bot a functionality to inject to the Prompt when a local state in Bot fits rather rather than holding the state here? Ideally Core should not be aware of local state in Bot.